i3pystatus/shell.py

```diff
@@ -14,9 +13,11 @@ class Shell(IntervalModule):
 
     color = "#FFFFFF"
     error_color = "#FF0000"
+    ignore_emtpy_stdout = False
```

Comment:
Damn symbol auto complete. Thanks for pointing this out @hcpl, making another PR now :)